teamcity: failed tests on master: test/TestPGUnwrapError

The following tests appear to have failed:

[#826964](https://teamcity.cockroachdb.com/viewLog.html?buildId=826964):


```
--- FAIL: test/TestPGUnwrapError (0.000s)
Test ended in panic.

------- Stdout: -------
W180809 22:10:38.900836 5981 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180809 22:10:38.913275 5981 server/server.go:812  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180809 22:10:38.932182 5981 server/config.go:492  [n?] 1 storage engine initialized
I180809 22:10:38.932957 5981 server/config.go:495  [n?] RocksDB cache size: 128 MiB
I180809 22:10:38.932999 5981 server/config.go:495  [n?] store 0: in-memory, size 0 B
I180809 22:10:38.948396 5981 server/node.go:372  [n?] **** cluster db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721 has been created
I180809 22:10:38.948443 5981 server/server.go:1387  [n?] **** add additional nodes by specifying --join=127.0.0.1:45685
I180809 22:10:38.948569 5981 gossip/gossip.go:363  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45685" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > 
I180809 22:10:38.949525 5981 storage/store.go:1501  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180809 22:10:38.949586 5981 server/node.go:461  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7043.00 p25=7043.00 p50=7043.00 p75=7043.00 p90=7043.00 pMax=7043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180809 22:10:38.949631 5981 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180809 22:10:38.952439 6230 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180809 22:10:38.955612 5981 server/node.go:684  [n1] connecting to gossip network to verify cluster ID...
I180809 22:10:38.955640 5981 server/node.go:709  [n1] node connected via gossip and verified as part of cluster "db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721"
I180809 22:10:38.955710 5981 server/node.go:533  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180809 22:10:38.956183 5981 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:10:38.956216 5981 server/server.go:1819  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180809 22:10:38.957971 6244 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:10:38.965266 5981 server/server.go:1524  [n1] starting https server at 127.0.0.1:42641
I180809 22:10:38.965304 5981 server/server.go:1525  [n1] starting grpc/postgres server at 127.0.0.1:45685
I180809 22:10:38.965318 5981 server/server.go:1526  [n1] advertising CockroachDB node at 127.0.0.1:45685
W180809 22:10:38.966767 6024 rpc/nodedialer/nodedialer.go:89  [split,n1,s1,r1/1:/M{in-ax},intExec=log-range-event] unable to connect to n0: unable to look up descriptor for node 0
E180809 22:10:38.970017 6230 storage/queue.go:778  [split,n1,s1,r1/1:/M{in-ax}] unable to split [n1,s1,r1/1:/M{in-ax}] at key "/System/\"\"": split at key /System/"" failed: 0 rows affected by log insertion; expected exactly one row affected.
I180809 22:10:38.995050 5804 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180809 22:10:38.997869 6257 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r3]
I180809 22:10:39.008706 6277 storage/replica_command.go:279  [split,n1,s1,r3/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r4]
I180809 22:10:39.017322 6284 storage/replica_command.go:279  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r5]
W180809 22:10:39.018255 6222 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=edd5b9cc key=/Table/SystemConfigSpan/Start rw=true pri=0.00534127 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533852638.999738340,0 orig=1533852638.999738340,0 max=1533852638.999738340,0 wto=false rop=false seq=12
I180809 22:10:39.020705 6216 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180809 22:10:39.027610 6259 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180809 22:10:39.038106 5808 storage/replica_command.go:279  [split,n1,s1,r5/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r6]
I180809 22:10:39.047751 6273 storage/replica_command.go:279  [split,n1,s1,r6/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r7]
I180809 22:10:39.054591 6340 storage/replica_command.go:279  [split,n1,s1,r7/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r8]
I180809 22:10:39.059011 6355 storage/replica_command.go:279  [split,n1,s1,r8/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r9]
I180809 22:10:39.083732 6386 storage/replica_command.go:279  [split,n1,s1,r9/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r10]
I180809 22:10:39.124068 6303 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180809 22:10:39.126027 6389 storage/replica_command.go:279  [split,n1,s1,r10/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r11]
I180809 22:10:39.130725 6421 storage/replica_command.go:279  [split,n1,s1,r11/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r12]
I180809 22:10:39.135703 6328 storage/replica_command.go:279  [split,n1,s1,r12/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r13]
I180809 22:10:39.140942 6395 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180809 22:10:39.141836 6452 storage/replica_command.go:279  [split,n1,s1,r13/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r14]
I180809 22:10:39.155884 6437 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180809 22:10:39.159524 5981 server/server.go:1608  [n1] done ensuring all necessary migrations have run
I180809 22:10:39.159565 5981 server/server.go:1611  [n1] serving sql connections
I180809 22:10:39.163949 3423 storage/replica_command.go:279  [split,n1,s1,r14/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r15]
I180809 22:10:39.167241 6463 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180809 22:10:39.167496 6465 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45685} Attrs: Locality: ServerVersion:2.0-11} ClusterID:db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721 StartedAt:1533852638955675119 LastUp:1533852638955675119}
I180809 22:10:39.169776 6484 storage/replica_command.go:279  [split,n1,s1,r15/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r16]
I180809 22:10:39.176016 6493 storage/replica_command.go:279  [split,n1,s1,r16/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r17]
I180809 22:10:39.191791 6384 storage/replica_command.go:279  [split,n1,s1,r17/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r18]
I180809 22:10:39.200431 6473 storage/replica_command.go:279  [split,n1,s1,r18/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r19]
I180809 22:10:39.206811 6570 storage/replica_command.go:279  [split,n1,s1,r19/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r20]
W180809 22:10:39.211480 6579 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "split" id=0a4c4753 key=/Local/Range/Table/21/RangeDescriptor rw=true pri=0.00790693 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533852639.206832712,0 orig=1533852639.206832712,0 max=1533852639.206832712,0 wto=false rop=false seq=1
I180809 22:10:39.213148 6583 storage/replica_command.go:279  [split,n1,s1,r20/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r21]
I180809 22:10:39.231993 5953 storage/replica_command.go:279  [split,n1,s1,r21/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r22]
I180809 22:10:39.238633 6502 storage/replica_command.go:279  [split,n1,s1,r22/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r23]
panic: test timed out after 8m0s

goroutine 6610 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1240 +0xfc
created by time.goFunc
	/usr/local/go/src/time/sleep.go:172 +0x44

goroutine 1 [chan receive, 7 minutes]:
testing.(*T).Run(0xc421557a40, 0x2429bec, 0x11, 0x2541b78, 0x8b9001)
	/usr/local/go/src/testing/testing.go:825 +0x301
testing.runTests.func1(0xc4201c14a0)
	/usr/local/go/src/testing/testing.go:1063 +0x64
testing.tRunner(0xc4201c14a0, 0xc4206afdd8)
	/usr/local/go/src/testing/testing.go:777 +0xd0
testing.runTests(0xc4202d2b60, 0x375d520, 0x2b, 0x2b, 0xbed34cd62d9dfc4e)
	/usr/local/go/src/testing/testing.go:1061 +0x2c4
testing.(*M).Run(0xc42059ab00, 0x0)
	/usr/local/go/src/testing/testing.go:978 +0x171
github.com/cockroachdb/cockroach/pkg/sql/pgwire_test.TestMain(0xc42059ab00)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/main_test.go:37 +0x80
main.main()
	_testmain.go:170 +0x151

goroutine 6 [syscall, 8 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:139 +0xa6
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.0
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41

goroutine 11 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.flushDaemon()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1171 +0xf1
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:590 +0xf4

goroutine 12 [chan receive, 8 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.signalFlusher()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:597 +0xab
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:591 +0x10c

goroutine 98 [select, 8 minutes, locked to thread]:
runtime.gopark(0x2545490, 0x0, 0x2417d5d, 0x6, 0x18, 0x1)
	/usr/local/go/src/runtime/proc.go:291 +0x11a
runtime.selectgo(0xc42046af50, 0xc420254060)
	/usr/local/go/src/runtime/select.go:392 +0xe50
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:549 +0x1f4
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2361 +0x1

goroutine 5981 [semacquire, 7 minutes]:
sync.runtime_Semacquire(0xc420e1c02c)
	/usr/local/go/src/runtime/sema.go:56 +0x39
sync.(*WaitGroup).Wait(0xc420e1c020)
	/usr/local/go/src/sync/waitgroup.go:129 +0x72
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc420e1c000, 0x2919760, 0xc42004e130)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:484 +0x2d9
github.com/cockroachdb/cockroach/pkg/sql/pgwire_test.TestPGUnwrapError(0xc421557a40)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwire_test.go:451 +0x3ed
testing.tRunner(0xc421557a40, 0x2541b78)
	/usr/local/go/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:824 +0x2e0

goroutine 4123 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c845a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 290 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4201a9d40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 291 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4201a9e60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 3507 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc421082ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2262 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c84ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 867 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d2360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6048 [semacquire, 7 minutes]:
sync.runtime_notifyListWait(0xc420e5de70, 0xc4000000f2)
	/usr/local/go/src/runtime/sema.go:510 +0x10b
sync.(*Cond).Wait(0xc420e5de60)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc420e5dd40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:654 +0x72
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:642 +0x418

goroutine 6603 [select, 7 minutes]:
database/sql.(*DB).connectionOpener(0xc420215860, 0x2919720, 0xc4212ccb00)
	/usr/local/go/src/database/sql/sql.go:935 +0x119
created by database/sql.OpenDB
	/usr/local/go/src/database/sql/sql.go:634 +0x178

goroutine 6033 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c850e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2263 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c84fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 866 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d2240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6604 [select, 7 minutes]:
database/sql.(*DB).connectionResetter(0xc420215860, 0x2919720, 0xc4212ccb00)
	/usr/local/go/src/database/sql/sql.go:948 +0x12a
created by database/sql.OpenDB
	/usr/local/go/src/database/sql/sql.go:635 +0x1ae

goroutine 1633 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e6240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 5984 [semacquire, 7 minutes]:
sync.runtime_notifyListWait(0xc4200cc7b0, 0xc400000000)
	/usr/local/go/src/runtime/sema.go:510 +0x10b
sync.(*Cond).Wait(0xc4200cc7a0)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc4200cc680)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:654 +0x72
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:642 +0x418

goroutine 1634 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e65a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6036 [select, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps.func1(0x25457f8, 0xc4216de000, 0xc4200aac40, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/hlc/hlc.go:185 +0x13e
created by github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps
	/go/src/github.com/cockroachdb/cockroach/pkg/util/hlc/hlc.go:178 +0xaf

goroutine 2917 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d3200)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2916 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d30e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 5371 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc42154ad80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 3508 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4210830e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4744 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e6b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4743 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e66c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6024 [chan send (nil chan), 7 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.runnerRequest.run(0x29197e0, 0xc4211b6ea0, 0xc4216ae340, 0xc42032bce0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:75 +0xa6
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x29197e0, 0xc420812030)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:89 +0x9a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4209fced0, 0xc420e1c000, 0xc4209fcec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:199 +0xe9
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:192 +0xad

goroutine 5370 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc42154ac60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4124 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c846c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6034 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c85200)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288
--- FAIL: test/TestPGUnwrapError (2.560s)
Test ended in panic.

------- Stdout: -------
W180809 22:10:38.900836 5981 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180809 22:10:38.913275 5981 server/server.go:812  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180809 22:10:38.932182 5981 server/config.go:492  [n?] 1 storage engine initialized
I180809 22:10:38.932957 5981 server/config.go:495  [n?] RocksDB cache size: 128 MiB
I180809 22:10:38.932999 5981 server/config.go:495  [n?] store 0: in-memory, size 0 B
I180809 22:10:38.948396 5981 server/node.go:372  [n?] **** cluster db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721 has been created
I180809 22:10:38.948443 5981 server/server.go:1387  [n?] **** add additional nodes by specifying --join=127.0.0.1:45685
I180809 22:10:38.948569 5981 gossip/gossip.go:363  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:45685" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > 
I180809 22:10:38.949525 5981 storage/store.go:1501  [n1,s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I180809 22:10:38.949586 5981 server/node.go:461  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=0, queries=0.00, writes=0.00, bytesPerReplica={p10=7043.00 p25=7043.00 p50=7043.00 p75=7043.00 p90=7043.00 pMax=7043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180809 22:10:38.949631 5981 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180809 22:10:38.952439 6230 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180809 22:10:38.955612 5981 server/node.go:684  [n1] connecting to gossip network to verify cluster ID...
I180809 22:10:38.955640 5981 server/node.go:709  [n1] node connected via gossip and verified as part of cluster "db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721"
I180809 22:10:38.955710 5981 server/node.go:533  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180809 22:10:38.956183 5981 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:10:38.956216 5981 server/server.go:1819  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180809 22:10:38.957971 6244 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:10:38.965266 5981 server/server.go:1524  [n1] starting https server at 127.0.0.1:42641
I180809 22:10:38.965304 5981 server/server.go:1525  [n1] starting grpc/postgres server at 127.0.0.1:45685
I180809 22:10:38.965318 5981 server/server.go:1526  [n1] advertising CockroachDB node at 127.0.0.1:45685
W180809 22:10:38.966767 6024 rpc/nodedialer/nodedialer.go:89  [split,n1,s1,r1/1:/M{in-ax},intExec=log-range-event] unable to connect to n0: unable to look up descriptor for node 0
E180809 22:10:38.970017 6230 storage/queue.go:778  [split,n1,s1,r1/1:/M{in-ax}] unable to split [n1,s1,r1/1:/M{in-ax}] at key "/System/\"\"": split at key /System/"" failed: 0 rows affected by log insertion; expected exactly one row affected.
I180809 22:10:38.995050 5804 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180809 22:10:38.997869 6257 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r3]
I180809 22:10:39.008706 6277 storage/replica_command.go:279  [split,n1,s1,r3/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r4]
I180809 22:10:39.017322 6284 storage/replica_command.go:279  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r5]
W180809 22:10:39.018255 6222 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=edd5b9cc key=/Table/SystemConfigSpan/Start rw=true pri=0.00534127 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533852638.999738340,0 orig=1533852638.999738340,0 max=1533852638.999738340,0 wto=false rop=false seq=12
I180809 22:10:39.020705 6216 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180809 22:10:39.027610 6259 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180809 22:10:39.038106 5808 storage/replica_command.go:279  [split,n1,s1,r5/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r6]
I180809 22:10:39.047751 6273 storage/replica_command.go:279  [split,n1,s1,r6/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r7]
I180809 22:10:39.054591 6340 storage/replica_command.go:279  [split,n1,s1,r7/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r8]
I180809 22:10:39.059011 6355 storage/replica_command.go:279  [split,n1,s1,r8/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r9]
I180809 22:10:39.083732 6386 storage/replica_command.go:279  [split,n1,s1,r9/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r10]
I180809 22:10:39.124068 6303 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
I180809 22:10:39.126027 6389 storage/replica_command.go:279  [split,n1,s1,r10/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r11]
I180809 22:10:39.130725 6421 storage/replica_command.go:279  [split,n1,s1,r11/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r12]
I180809 22:10:39.135703 6328 storage/replica_command.go:279  [split,n1,s1,r12/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r13]
I180809 22:10:39.140942 6395 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180809 22:10:39.141836 6452 storage/replica_command.go:279  [split,n1,s1,r13/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r14]
I180809 22:10:39.155884 6437 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180809 22:10:39.159524 5981 server/server.go:1608  [n1] done ensuring all necessary migrations have run
I180809 22:10:39.159565 5981 server/server.go:1611  [n1] serving sql connections
I180809 22:10:39.163949 3423 storage/replica_command.go:279  [split,n1,s1,r14/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r15]
I180809 22:10:39.167241 6463 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180809 22:10:39.167496 6465 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:45685} Attrs: Locality: ServerVersion:2.0-11} ClusterID:db9bcb07-8d8d-46e3-b3a7-ba6c5ffc8721 StartedAt:1533852638955675119 LastUp:1533852638955675119}
I180809 22:10:39.169776 6484 storage/replica_command.go:279  [split,n1,s1,r15/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r16]
I180809 22:10:39.176016 6493 storage/replica_command.go:279  [split,n1,s1,r16/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r17]
I180809 22:10:39.191791 6384 storage/replica_command.go:279  [split,n1,s1,r17/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r18]
I180809 22:10:39.200431 6473 storage/replica_command.go:279  [split,n1,s1,r18/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r19]
I180809 22:10:39.206811 6570 storage/replica_command.go:279  [split,n1,s1,r19/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r20]
W180809 22:10:39.211480 6579 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "split" id=0a4c4753 key=/Local/Range/Table/21/RangeDescriptor rw=true pri=0.00790693 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533852639.206832712,0 orig=1533852639.206832712,0 max=1533852639.206832712,0 wto=false rop=false seq=1
I180809 22:10:39.213148 6583 storage/replica_command.go:279  [split,n1,s1,r20/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r21]
I180809 22:10:39.231993 5953 storage/replica_command.go:279  [split,n1,s1,r21/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r22]
I180809 22:10:39.238633 6502 storage/replica_command.go:279  [split,n1,s1,r22/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r23]
panic: test timed out after 8m0s

goroutine 6610 [running]:
testing.(*M).startAlarm.func1()
	/usr/local/go/src/testing/testing.go:1240 +0xfc
created by time.goFunc
	/usr/local/go/src/time/sleep.go:172 +0x44

goroutine 1 [chan receive, 7 minutes]:
testing.(*T).Run(0xc421557a40, 0x2429bec, 0x11, 0x2541b78, 0x8b9001)
	/usr/local/go/src/testing/testing.go:825 +0x301
testing.runTests.func1(0xc4201c14a0)
	/usr/local/go/src/testing/testing.go:1063 +0x64
testing.tRunner(0xc4201c14a0, 0xc4206afdd8)
	/usr/local/go/src/testing/testing.go:777 +0xd0
testing.runTests(0xc4202d2b60, 0x375d520, 0x2b, 0x2b, 0xbed34cd62d9dfc4e)
	/usr/local/go/src/testing/testing.go:1061 +0x2c4
testing.(*M).Run(0xc42059ab00, 0x0)
	/usr/local/go/src/testing/testing.go:978 +0x171
github.com/cockroachdb/cockroach/pkg/sql/pgwire_test.TestMain(0xc42059ab00)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/main_test.go:37 +0x80
main.main()
	_testmain.go:170 +0x151

goroutine 6 [syscall, 8 minutes]:
os/signal.signal_recv(0x0)
	/usr/local/go/src/runtime/sigqueue.go:139 +0xa6
os/signal.loop()
	/usr/local/go/src/os/signal/signal_unix.go:22 +0x22
created by os/signal.init.0
	/usr/local/go/src/os/signal/signal_unix.go:28 +0x41

goroutine 11 [chan receive]:
github.com/cockroachdb/cockroach/pkg/util/log.flushDaemon()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1171 +0xf1
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:590 +0xf4

goroutine 12 [chan receive, 8 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.signalFlusher()
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:597 +0xab
created by github.com/cockroachdb/cockroach/pkg/util/log.init.0
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:591 +0x10c

goroutine 98 [select, 8 minutes, locked to thread]:
runtime.gopark(0x2545490, 0x0, 0x2417d5d, 0x6, 0x18, 0x1)
	/usr/local/go/src/runtime/proc.go:291 +0x11a
runtime.selectgo(0xc42046af50, 0xc420254060)
	/usr/local/go/src/runtime/select.go:392 +0xe50
runtime.ensureSigM.func1()
	/usr/local/go/src/runtime/signal_unix.go:549 +0x1f4
runtime.goexit()
	/usr/local/go/src/runtime/asm_amd64.s:2361 +0x1

goroutine 5981 [semacquire, 7 minutes]:
sync.runtime_Semacquire(0xc420e1c02c)
	/usr/local/go/src/runtime/sema.go:56 +0x39
sync.(*WaitGroup).Wait(0xc420e1c020)
	/usr/local/go/src/sync/waitgroup.go:129 +0x72
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).Stop(0xc420e1c000, 0x2919760, 0xc42004e130)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:484 +0x2d9
github.com/cockroachdb/cockroach/pkg/sql/pgwire_test.TestPGUnwrapError(0xc421557a40)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgwire_test.go:451 +0x3ed
testing.tRunner(0xc421557a40, 0x2541b78)
	/usr/local/go/src/testing/testing.go:777 +0xd0
created by testing.(*T).Run
	/usr/local/go/src/testing/testing.go:824 +0x2e0

goroutine 4123 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c845a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 290 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4201a9d40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 291 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4201a9e60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 3507 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc421082ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2262 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c84ea0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 867 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d2360)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6048 [semacquire, 7 minutes]:
sync.runtime_notifyListWait(0xc420e5de70, 0xc4000000f2)
	/usr/local/go/src/runtime/sema.go:510 +0x10b
sync.(*Cond).Wait(0xc420e5de60)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc420e5dd40)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:654 +0x72
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:642 +0x418

goroutine 6603 [select, 7 minutes]:
database/sql.(*DB).connectionOpener(0xc420215860, 0x2919720, 0xc4212ccb00)
	/usr/local/go/src/database/sql/sql.go:935 +0x119
created by database/sql.OpenDB
	/usr/local/go/src/database/sql/sql.go:634 +0x178

goroutine 6033 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c850e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2263 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c84fc0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 866 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d2240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6604 [select, 7 minutes]:
database/sql.(*DB).connectionResetter(0xc420215860, 0x2919720, 0xc4212ccb00)
	/usr/local/go/src/database/sql/sql.go:948 +0x12a
created by database/sql.OpenDB
	/usr/local/go/src/database/sql/sql.go:635 +0x1ae

goroutine 1633 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e6240)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 5984 [semacquire, 7 minutes]:
sync.runtime_notifyListWait(0xc4200cc7b0, 0xc400000000)
	/usr/local/go/src/runtime/sema.go:510 +0x10b
sync.(*Cond).Wait(0xc4200cc7a0)
	/usr/local/go/src/sync/cond.go:56 +0x80
github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).syncLoop(0xc4200cc680)
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:654 +0x72
created by github.com/cockroachdb/cockroach/pkg/storage/engine.(*RocksDB).open
	/go/src/github.com/cockroachdb/cockroach/pkg/storage/engine/rocksdb.go:642 +0x418

goroutine 1634 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e65a0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6036 [select, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps.func1(0x25457f8, 0xc4216de000, 0xc4200aac40, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/hlc/hlc.go:185 +0x13e
created by github.com/cockroachdb/cockroach/pkg/util/hlc.(*Clock).StartMonitoringForwardClockJumps
	/go/src/github.com/cockroachdb/cockroach/pkg/util/hlc/hlc.go:178 +0xaf

goroutine 2917 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d3200)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 2916 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4205d30e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 5371 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc42154ad80)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 3508 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4210830e0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4744 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e6b40)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4743 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc4200e66c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6024 [chan send (nil chan), 7 minutes]:
github.com/cockroachdb/cockroach/pkg/sql.runnerRequest.run(0x29197e0, 0xc4211b6ea0, 0xc4216ae340, 0xc42032bce0, 0x0, 0x0)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:75 +0xa6
github.com/cockroachdb/cockroach/pkg/sql.(*DistSQLPlanner).initRunners.func1(0x29197e0, 0xc420812030)
	/go/src/github.com/cockroachdb/cockroach/pkg/sql/distsql_running.go:89 +0x9a
github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker.func1(0xc4209fced0, 0xc420e1c000, 0xc4209fcec0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:199 +0xe9
created by github.com/cockroachdb/cockroach/pkg/util/stop.(*Stopper).RunWorker
	/go/src/github.com/cockroachdb/cockroach/pkg/util/stop/stopper.go:192 +0xad

goroutine 5370 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc42154ac60)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 4124 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c846c0)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288

goroutine 6034 [chan receive, 7 minutes]:
github.com/cockroachdb/cockroach/pkg/util/log.(*loggingT).gcDaemon(0xc420c85200)
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/clog.go:1223 +0x68
created by github.com/cockroachdb/cockroach/pkg/util/log.NewSecondaryLogger
	/go/src/github.com/cockroachdb/cockroach/pkg/util/log/secondary_log.go:80 +0x288
------- Stdout: -------
W180809 22:20:18.294692 6198 server/status/runtime.go:287  [n?] Could not parse build timestamp: parsing time "" as "2006/01/02 15:04:05": cannot parse "" as "2006"
I180809 22:20:18.416635 6198 server/server.go:812  [n?] monitoring forward clock jumps based on server.clock.forward_jump_check_enabled
I180809 22:20:18.428985 6198 server/config.go:492  [n?] 1 storage engine initialized
I180809 22:20:18.429147 6198 server/config.go:495  [n?] RocksDB cache size: 128 MiB
I180809 22:20:18.429254 6198 server/config.go:495  [n?] store 0: in-memory, size 0 B
I180809 22:20:18.470505 6198 util/stop/stopper.go:537  [n?] quiescing; tasks left:
1      [async] storage.pendingLeaseRequest: requesting lease
I180809 22:20:18.474683 6198 server/node.go:372  [n?] **** cluster 8df183b0-d85e-4957-8f35-cb3952574c62 has been created
I180809 22:20:18.474874 6198 server/server.go:1387  [n?] **** add additional nodes by specifying --join=127.0.0.1:43101
I180809 22:20:18.475832 6198 gossip/gossip.go:363  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:43101" > attrs:<> locality:<> ServerVersion:<major_val:2 minor_val:0 patch:0 unstable:11 > 
I180809 22:20:18.518463 6198 server/node.go:461  [n1] initialized store [n1,s1]: disk (capacity=512 MiB, available=512 MiB, used=0 B, logicalBytes=6.9 KiB), ranges=1, leases=1, queries=0.00, writes=0.00, bytesPerReplica={p10=7043.00 p25=7043.00 p50=7043.00 p75=7043.00 p90=7043.00 pMax=7043.00}, writesPerReplica={p10=0.00 p25=0.00 p50=0.00 p75=0.00 p90=0.00 pMax=0.00}
I180809 22:20:18.519459 6198 storage/stores.go:222  [n1] read 0 node addresses from persistent storage
I180809 22:20:18.520624 6198 server/node.go:684  [n1] connecting to gossip network to verify cluster ID...
I180809 22:20:18.520986 6198 server/node.go:709  [n1] node connected via gossip and verified as part of cluster "8df183b0-d85e-4957-8f35-cb3952574c62"
I180809 22:20:18.522113 6198 server/node.go:533  [n1] node=1: started with [<no-attributes>=<in-mem>] engine(s) and attributes []
I180809 22:20:18.527970 6198 server/status/recorder.go:652  [n1] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:20:18.528445 6198 server/server.go:1819  [n1] Could not start heap profiler worker due to: directory to store profiles could not be determined
I180809 22:20:18.530671 6198 server/server.go:1524  [n1] starting https server at 127.0.0.1:33683
I180809 22:20:18.537731 6198 server/server.go:1525  [n1] starting grpc/postgres server at 127.0.0.1:43101
I180809 22:20:18.537867 6198 server/server.go:1526  [n1] advertising CockroachDB node at 127.0.0.1:43101
I180809 22:20:18.532250 6484 server/status/recorder.go:652  [n1,summaries] available memory from cgroups (8.0 EiB) exceeds system memory 16 GiB, using system memory
I180809 22:20:18.571793 6474 storage/replica_command.go:279  [split,n1,s1,r1/1:/M{in-ax}] initiating a split of this range at key /System/"" [r2]
I180809 22:20:18.702933 6261 storage/replica_command.go:279  [split,n1,s1,r2/1:/{System/-Max}] initiating a split of this range at key /System/NodeLiveness [r3]
I180809 22:20:18.839750 6093 sql/event_log.go:126  [n1,intExec=optInToDiagnosticsStatReporting] Event: "set_cluster_setting", target: 0, info: {SettingName:diagnostics.reporting.enabled Value:true User:root}
I180809 22:20:18.985983 6461 storage/replica_command.go:279  [split,n1,s1,r3/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/NodeLivenessMax [r4]
I180809 22:20:19.103367 6238 storage/replica_command.go:279  [split,n1,s1,r4/1:/{System/NodeL…-Max}] initiating a split of this range at key /System/tsd [r5]
I180809 22:20:19.181538 6531 storage/replica_command.go:279  [split,n1,s1,r5/1:/{System/tsd-Max}] initiating a split of this range at key /System/"tse" [r6]
I180809 22:20:19.311103 6518 storage/replica_command.go:279  [split,n1,s1,r6/1:/{System/tse-Max}] initiating a split of this range at key /Table/SystemConfigSpan/Start [r7]
I180809 22:20:19.360037 6514 sql/event_log.go:126  [n1,intExec=set-setting] Event: "set_cluster_setting", target: 0, info: {SettingName:version Value:$1 User:root}
I180809 22:20:19.428637 6591 storage/replica_command.go:279  [split,n1,s1,r7/1:/{Table/System…-Max}] initiating a split of this range at key /Table/11 [r8]
W180809 22:20:19.504816 6588 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "sql txn" id=d91a2b89 key=/Table/SystemConfigSpan/Start rw=true pri=0.00343681 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533853219.369991413,0 orig=1533853219.369991413,0 max=1533853219.369991413,0 wto=false rop=false seq=6
I180809 22:20:19.546566 6583 sql/event_log.go:126  [n1,intExec=disableNetTrace] Event: "set_cluster_setting", target: 0, info: {SettingName:trace.debug.enable Value:false User:root}
I180809 22:20:19.598562 6503 storage/replica_command.go:279  [split,n1,s1,r8/1:/{Table/11-Max}] initiating a split of this range at key /Table/12 [r9]
I180809 22:20:19.630653 6527 rpc/nodedialer/nodedialer.go:92  [replica consistency checker,n1,s1,r1/1:/{Min-System/}] connection to n1 established
I180809 22:20:19.670807 6553 storage/replica_command.go:279  [split,n1,s1,r9/1:/{Table/12-Max}] initiating a split of this range at key /Table/13 [r10]
I180809 22:20:19.741343 6659 storage/replica_command.go:279  [split,n1,s1,r10/1:/{Table/13-Max}] initiating a split of this range at key /Table/14 [r11]
I180809 22:20:19.792668 6609 storage/replica_command.go:279  [split,n1,s1,r11/1:/{Table/14-Max}] initiating a split of this range at key /Table/15 [r12]
I180809 22:20:19.829754 6646 sql/event_log.go:126  [n1,intExec=initializeClusterSecret] Event: "set_cluster_setting", target: 0, info: {SettingName:cluster.secret Value:gen_random_uuid()::STRING User:root}
W180809 22:20:19.867640 6706 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "split" id=4dd7d05f key=/Local/Range/Table/14/RangeDescriptor rw=true pri=0.00401163 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533853219.792760283,0 orig=1533853219.792760283,0 max=1533853219.792760283,0 wto=false rop=false seq=1
W180809 22:20:19.871691 6665 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "split" id=4dd7d05f key=/Local/Range/Table/14/RangeDescriptor rw=true pri=0.00401163 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533853219.792760283,0 orig=1533853219.792760283,0 max=1533853219.792760283,0 wto=false rop=false seq=1
I180809 22:20:19.914991 6616 storage/replica_command.go:279  [split,n1,s1,r12/1:/{Table/15-Max}] initiating a split of this range at key /Table/16 [r13]
I180809 22:20:19.979597 6713 storage/replica_command.go:279  [split,n1,s1,r13/1:/{Table/16-Max}] initiating a split of this range at key /Table/17 [r14]
I180809 22:20:20.004523 6537 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 50, info: {DatabaseName:defaultdb Statement:CREATE DATABASE IF NOT EXISTS defaultdb User:root}
I180809 22:20:20.067694 6572 storage/replica_command.go:279  [split,n1,s1,r14/1:/{Table/17-Max}] initiating a split of this range at key /Table/18 [r15]
I180809 22:20:20.086857 6621 sql/event_log.go:126  [n1,intExec=create-default-db] Event: "create_database", target: 51, info: {DatabaseName:postgres Statement:CREATE DATABASE IF NOT EXISTS postgres User:root}
I180809 22:20:20.135762 6198 server/server.go:1608  [n1] done ensuring all necessary migrations have run
I180809 22:20:20.135887 6198 server/server.go:1611  [n1] serving sql connections
I180809 22:20:20.155633 6679 server/server_update.go:67  [n1] no need to upgrade, cluster already at the newest version
I180809 22:20:20.174136 6681 sql/event_log.go:126  [n1] Event: "node_join", target: 1, info: {Descriptor:{NodeID:1 Address:{NetworkField:tcp AddressField:127.0.0.1:43101} Attrs: Locality: ServerVersion:2.0-11} ClusterID:8df183b0-d85e-4957-8f35-cb3952574c62 StartedAt:1533853218521041501 LastUp:1533853218521041501}
I180809 22:20:20.182708 6577 storage/replica_command.go:279  [split,n1,s1,r15/1:/{Table/18-Max}] initiating a split of this range at key /Table/19 [r16]
I180809 22:20:20.227831 6748 storage/replica_command.go:279  [split,n1,s1,r16/1:/{Table/19-Max}] initiating a split of this range at key /Table/20 [r17]
W180809 22:20:20.262421 6751 storage/intent_resolver.go:665  [n1,s1] failed to push during intent resolution: failed to push "split" id=507a6a17 key=/Local/Range/Table/19/RangeDescriptor rw=true pri=0.00622738 iso=SERIALIZABLE stat=PENDING epo=0 ts=1533853220.253156789,1 orig=1533853220.227921254,0 max=1533853220.227921254,0 wto=false rop=false seq=1
I180809 22:20:20.276995 6672 storage/replica_command.go:279  [split,n1,s1,r17/1:/{Table/20-Max}] initiating a split of this range at key /Table/21 [r18]
I180809 22:20:20.320724 6820 storage/replica_command.go:279  [split,n1,s1,r18/1:/{Table/21-Max}] initiating a split of this range at key /Table/22 [r19]
I180809 22:20:20.377703 6790 storage/replica_command.go:279  [split,n1,s1,r19/1:/{Table/22-Max}] initiating a split of this range at key /Table/23 [r20]
I180809 22:20:20.415593 6814 storage/replica_command.go:279  [split,n1,s1,r20/1:/{Table/23-Max}] initiating a split of this range at key /Table/50 [r21]
I180809 22:20:20.479373 6882 storage/replica_command.go:279  [split,n1,s1,r21/1:/{Table/50-Max}] initiating a split of this range at key /Table/51 [r22]
I180809 22:20:20.653254 6198 util/stop/stopper.go:537  quiescing; tasks left:
1      [async] transport racer
I180809 22:20:20.706215 6090 kv/transport_race.go:67  transport race promotion: ran 87 iterations on up to 706 requests

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/server/node.go
+++ b/pkg/server/node.go
@@ -383,6 +383,8 @@ func (n *Node) onClusterVersionChange(cv cluster.ClusterVersion) {
 // start starts the node by registering the storage instance for the
 // RPC service "Node" and initializing stores for each specified
 // engine. Launches periodic store gossiping in a goroutine.
+// A callback can be optionally provided that will be invoked once this node's
+// NodeDescriptor is available, to help bootstrapping.
 func (n *Node) start(
 	ctx context.Context,
 	addr net.Addr,
@@ -390,6 +392,7 @@ func (n *Node) start(
 	attrs roachpb.Attributes,
 	locality roachpb.Locality,
 	cv cluster.ClusterVersion,
+	nodeDescriptorCallback func(descriptor roachpb.NodeDescriptor),
 ) error {
 	n.storeCfg.Settings.Version.OnChange(n.onClusterVersionChange)
 	if err := n.storeCfg.Settings.InitializeVersion(cv); err != nil {
@@ -430,6 +433,12 @@ func (n *Node) start(
 		Attrs:         attrs,
 		Locality:      locality,
 		ServerVersion: n.storeCfg.Settings.Version.ServerVersion,
+	}
+	// Invoke any passed in nodeDescriptorCallback as soon as it's available, to
+	// ensure that other components (currently the DistSQLPlanner) are initialized
+	// before store startup continues.
+	if nodeDescriptorCallback != nil {
+		nodeDescriptorCallback(n.Descriptor)
 	}
 
 	// Gossip the node descriptor to make this node addressable by node ID.

--- a/pkg/server/node_test.go
+++ b/pkg/server/node_test.go
@@ -170,7 +170,8 @@ func createAndStartTestNode(
 		t.Fatal(err)
 	}
 	if err := node.start(context.Background(), addr, bootstrappedEngines, newEngines,
-		roachpb.Attributes{}, locality, cv); err != nil {
+		roachpb.Attributes{}, locality, cv, nil, /*nodeDescriptorCallback */
+	); err != nil {
 		t.Fatal(err)
 	}
 	if err := WaitForInitialSplits(node.storeCfg.DB); err != nil {
@@ -412,6 +413,7 @@ func TestCorruptedClusterID(t *testing.T) {
 	if err := node.start(
 		context.Background(), serverAddr, bootstrappedEngines, newEngines,
 		roachpb.Attributes{}, roachpb.Locality{}, cv,
+		nil, /* nodeDescriptorCallback */
 	); !testutils.IsError(err, "unidentified store") {
 		t.Errorf("unexpected error %v", err)
 	}

--- a/pkg/server/server.go
+++ b/pkg/server/server.go
@@ -1438,6 +1438,7 @@ If problems persist, please see ` + base.DocsURL("cluster-setup-troubleshooting.
 		s.cfg.NodeAttributes,
 		s.cfg.Locality,
 		cv,
+		s.execCfg.DistSQLPlanner.SetNodeDesc,
 	); err != nil {
 		return err
 	}
@@ -1449,8 +1450,6 @@ If problems persist, please see ` + base.DocsURL("cluster-setup-troubleshooting.
 		},
 		time.NewTicker,
 	)
-
-	s.execCfg.DistSQLPlanner.SetNodeDesc(s.node.Descriptor)
 
 	// Cluster ID should have been determined by this point.
 	if s.rpcContext.ClusterID.Get() == uuid.Nil {

